database/seeders/DatabaseSeeder.php

```diff
@@ -3,7 +3,6 @@
 namespace Database\Seeders;
 
 use App\Models\Company;
-use App\Models\Employee;
```

Comment:
Don't you use Employee seeder?

Reply:
No...im using it inside the bank account because i wanted to create an employee with a bank account @Raza9798 
![image](https://github.com/user-attachments/assets/086adba8-a440-4f06-9cc0-9e2e210d22c0)

Reply:
So are you creating the Employees inside the BankAccountFactory?

Reply:
Yess...employee and bank account has a one - to - one relationship

Reply:
you have sperate seeders and factories for each module right?

Reply:
I have separate factories for each module and one seeder like this

![image](https://github.com/user-attachments/assets/7958305b-75ab-4968-a223-3015f6c4b01f)